src/component/MainTodo/MainTodo.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { MainList } from './MainList';
+
+interface Props {
+  formValue: Todo[];
+}
+
+export const MainTodo: React.FC<Props> = ({ formValue }) => {
+  return (
+    <section className="todoapp__main">
+      {formValue.map((formList) => (
+        <MainList
+          formList={formList}
+          key={formList.id}
+        />
```

Comment:
For now you map over a value to render a list? I think it should be the other way around